### PR TITLE
Distortion global

### PR DESCRIPTION
**Pull request by lcalvet**
Allows optimization of the radial distortion in globalSfM.
Distortion coefficients initialized as in (incremental)SfM.

### CODE REVIEW

File: src/software/SfM/SfMIOHelper.hpp [hunk 1]
```diff
@@ -34,10 +34,13 @@ struct IntrinsicCameraInfo
   size_t m_w, m_h;
   float m_focal;
   Mat3 m_K;
+  double m_k1;
```
Comment (pmoulon) [inline src/software/SfM/SfMIOHelper.hpp]: Does this modification is useful ?

**Comment by pmoulon** [comment]
Thanks @lcalvet for this pull request.
Don't take care about travis failure.

Any feedback about it?

File: src/software/SfM/SfMIOHelper.hpp (hunk 1 above)
Comment (lcalvet) [inline src/software/SfM/SfMIOHelper.hpp]: Yes, required.

**Comment by pmoulon** [comment]
@lcalvet Pull request accepted. :+1: 
I have checked on https://github.com/openMVG/SfM_quality_evaluation and pose estimation remains stable. Sometimes it seems that angular error to GT is a bit better.

**Pull request merged**